x/slashing/client/rest/query.go

```diff
@@ -45,8 +45,8 @@ func signingInfoHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 		}
 
 		params := types.NewQuerySigningInfoParams(sdk.ConsAddress(pk.Address()))
+		bz, err := params.Marshal()
 
-		bz, err := cliCtx.Codec.MarshalJSON(params)
```

Comment:
Why are we changing serialization? We should still use `cliCtx.Codec.MarshalJSON(params)`